### ISSUE TITLE
pretrained hypernetwork model

### ISSUE THREAD

**Issue by SlZeroth** [KohakuBlueleaf/HyperKohaku#9]
first thank you for progress awesome project !

I want to know about milestone or progress of this project

did you train hyper network model now? or

how many time take to train hyper network model?

Finally, based on the current state, I wonder if the current project is actually usable.

**Comment by KohakuBlueleaf** [comment]
Still training……
QQ

**Comment by KohakuBlueleaf** [comment]
It should be usable
But "how to get reasonable result" still need more research

**Comment by sasso-effe** [comment]
> Still training……
@KohakuBlueleaf are you still training the loras or are you already training HyperDreamBooth?

**Comment by KohakuBlueleaf** [comment]
> > Still training……
> 
> @KohakuBlueleaf are you still training the loras or are you already training HyperDreamBooth?

Hyperdreambooth

You can actually skip stage II
Directly train the hypernetwork is doable
Already tried on LyCORIS repo

The problem is I need someone to help me to implement something like "mask face" trick for train on face
Or something like that

I have not much experience about DreamBooth so cannot get a good training method for pre-optimized loras

**Comment by sasso-effe** [comment]
If you skip stage 1, what do you use as ground truth weights during the hypernetwork training (Figure 2, phase 1 in the paper)? Do you randomly initialize them? I am trying to understand it from your code but I am a bit lost

**Comment by KohakuBlueleaf** [comment]
> If you skip stage 1, what do you use as ground truth weights during the hypernetwork training (Figure 2, phase 1 in the paper)? Do you randomly initialize them? I am trying to understand it from your code but I am a bit lost

Actually you only need the diffusion loss
Weight loss is jot necessary
But it will give you better ( if your pre-optimized weights is good) result

**Comment by garychan22** [comment]
@KohakuBlueleaf  If the stage 1 is skipped, should we set the add_constant flag to True in the HyperDream?

**Comment by KohakuBlueleaf** [comment]
> @KohakuBlueleaf If the stage 1 is skipped, should we set the add_constant flag to True in the HyperDream?

I think yes
But I am not sure if it will work
Will check my code

**Comment by nullarch** [comment]
@KohakuBlueleaf Hi is this project stop now?

**Comment by KohakuBlueleaf** [comment]
> @KohakuBlueleaf Hi is this project stop now?

Not, but I just don't have time to let it keep moving...
I have literally over 10 proj on hand....

**Comment by nullarch** [comment]
Thank you for your hard work. Is the hyperdreambooth project a low priority among the 10 ongoing projects? @KohakuBlueleaf

**Comment by KohakuBlueleaf** [comment]
> Thank you for your hard work. Is the hyperdreambooth project a low priority among the 10 ongoing projects? @KohakuBlueleaf

Yes it is relatively low
